tmt/base.py

```diff
@@ -3859,7 +3859,7 @@ def runs(self) -> bool:
         if keep is not None:
             # Sort by modify time of the workdirs and keep the newest workdirs
             all_workdirs.sort(
-                key=lambda workdir: os.path.getmtime(workdir / 'run.yaml'), reverse=True)
+                key=lambda workdir: os.path.getatime(workdir), reverse=True)
```

Comment:
* Is the change `getmtime` -> `getatime` intentional? If so, why? Can you add a comment on why `atime` is better than `mtime`?
* seems like a place where `os.path` prevailed while we moved to `pathlib`. Since you're changing the line, can you switch it to use `Path` instead? Should be something like `workdir.stat().st_mtime` (or `st_atime`).